admin_manual/enterprise/external_storage/windows-network-drive_configuration.rst

```diff
@@ -264,6 +270,34 @@ See `Configuring wnd:listen to run as a service
 in the documentation wiki for tips on running the listener as a service via
 cron, and by creating a `systemd`_ startup script.
 
+
+.. _password-options-label:
+
+Password Options
+----------------
+
+There are three ways to supply a password:
+
+#. Interactively in response to a password prompt.
+#. Sent as a parameter to the command, e.g., ``occ wnd:listen host share username password``.
+#. Read from a file, using the ``--password-file`` switch to specify the file to read from. 
+#. Using 3rd party software to store and fetch the password. When using this option, the 3rd party app needs to show the password as plaintext on standard output.
```

Comment:
I'd add the couple of examples in the PR so they know how they can integrate other tools. As it is right now it isn't clear how they can achieve that.